rules/prefer-math-trunc.js

```diff
@@ -3,27 +3,33 @@ const getDocumentationUrl = require('./utils/get-documentation-url');
 
 const MESSAGE_ID_BITWISE_OR = 'bitwiseOr';
 const MESSAGE_ID_BITWISE_NOT = 'bitwiseNot';
+const MESSAGE_ID_BITWISE_SIGNED_RIGHT_SHIFT = 'bitwiseSignedRightShift';
+const MESSAGE_ID_BITWISE_LEFT_SHIFT = 'bitwiseLeftShift';
+const MESSAGE_ID_BITWISE_XOR = 'bitwiseXor';
+const MESSAGE_ID_BITWISE_AND = 'bitwiseAnd';
 const messages = {
 	[MESSAGE_ID_BITWISE_OR]: 'Use `Math.trunc` instead of `| 0`.',
-	[MESSAGE_ID_BITWISE_NOT]: 'Use `Math.trunc` instead of `~~`.'
+	[MESSAGE_ID_BITWISE_NOT]: 'Use `Math.trunc` instead of `~~`.',
+	[MESSAGE_ID_BITWISE_SIGNED_RIGHT_SHIFT]: 'Use `Math.trunc` instead of `>> 0`.',
+	[MESSAGE_ID_BITWISE_LEFT_SHIFT]: 'Use `Math.trunc` instead of `<< 0`.',
+	[MESSAGE_ID_BITWISE_XOR]: 'Use `Math.trunc` instead of `^ 0`.',
+	[MESSAGE_ID_BITWISE_AND]: 'Use `Math.trunc` instead of `& 0xF`.'
```

Comment:
No need use so many messageIds,

```js
[MESSAGE_ID_BITWISE]: 'Use Use `Math.trunc` instead of `{{operator}} {{value}}`'
```

```js
context.report({
	messageId: MESSAGE_ID_BITWISE,
	data: {
		operator: '&',
		value: '0xFFFFF'
	}
});
```

Reply:
Ohh I didn't know that! It will be way better